Makefile

```diff
@@ -23,7 +23,7 @@ else
 	DO_BAZ=eval
 endif
 # x86_64 aarch64 crossbuild-aarch64
-BUILD_ARCH?=x86_64
+BUILD_ARCH?=$(shell uname -m)
```

Comment:
Hmm wouldn't this be an issue if you're on a mac and doing x86 testing? @mhenriks IIRC that's your setup?